### PR TITLE
Integrate monitoring into the hosting bundle

### PR DESCRIPTION
**Pull request by apfelbox**
| Q       | A
| ------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Docs PR | no docs yet

This just replaces the monitoring bundle. We should always have the full suite of functionality available.

+ adds Github infrastructure

### CODE REVIEW

**Comment by apfelbox** [comment]
This is for a v2 release.

File: src/Twig/MonitoringTwigExtension.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        {
+            return "";
+        }
+
+        $assetUrl = null !== $this->assetHelper
+            ? $this->assetHelper->getUrl("@hosting/js/trackjs.js")
+            : $this->packages->getUrl("bundles/becklynhosting/js/trackjs.js");
+
+        return \sprintf(
+            '<script src="%s"></script><script>window.TrackJS && TrackJS.install(%s)</script>',
```
Comment (keichinger) [inline src/Twig/MonitoringTwigExtension.php:84]: Consistency: `window.TrackJS && window.TrackJS.install(%s)`?

Comment (apfelbox) [reply src/Twig/MonitoringTwigExtension.php:84]: Just copied:

2019-01-16 at 16 52

Comment (keichinger) [reply src/Twig/MonitoringTwigExtension.php:84]: Alright :D If they insist, we keep it like that :P

**Review by keichinger — APPROVED** [review]

**Pull request merged**